docs/DeveloperGuide.md

```diff
@@ -296,33 +296,141 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 *{More to be added}*
 
 ### Use cases
-
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
-
-**Use case: Delete a person**
-
-**MSS**
-
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-
-    Use case ends.
-
-**Extensions**
-
-* 2a. The list is empty.
-
-  Use case ends.
-
-* 3a. The given index is invalid.
-
-    * 3a1. AddressBook shows an error message.
-
-      Use case resumes at step 2.
-
-*{More to be added}*
+(For all use cases below, the **System** is the `WedLog` and the **Actor** is the `user`, unless specified otherwise)
+
+>Use case: UC1 - Add a guest
+
+**MSS:**
+1. User provides guest details.
+2. WedLog adds the guest to the guest list.
+<br>Use case ends.
+
+**Extensions:**
+<br>1a. The given input format is invalid.
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;1a1. WedLog shows an error message.</span>
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;1a2. User provides guest details in a different format.</span>
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;Steps 1a1-1a2 are repeated until the User input format is valid.</span>
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;Use case resumes from step 2.</span>
+<br>1b. The guest list has reached maximum capacity.
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;1b1. WedLog shows an error message.</span>
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;Use case ends.</span>
+<br>1c. The guest already exists in the guest list.
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;1c1. WedLog shows an error message.</span>
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;Use case ends.</span>
+<br>*a. At any time, user inputs an invalid command/syntax
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;a1. WedLog shows an error message.</span>
+
+>Use case: UC2 - Delete a guest
+
+**MSS:**
+1. User requests to list all guests
+2. WedLog shows a list of guests
+3. User requests to delete a specific guest in the list via its index
+4. WedLog deletes the guest
+   <br>Use case ends.
+
+**Extensions:**
+<br>3a. The given index is invalid.
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;3a1. WedLog shows an error message.</span>
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;3a2. User requests to delete the guest using a different index</span>
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;Steps 3a1-3a2 are repeated until the User provides a valid index.</span>
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;Use case resumes from step 4.</span>
+<br>*a. At any time, user inputs an invalid command/syntax
+<br><span>&nbsp;&nbsp;&nbsp;&nbsp;a1. WedLog shows an error message.</span>
+
+>Use case: UC3 - VIew all guests
+
+**MSS:**
+1. User requests to list all guests
+2. WedLog displays a list of guests
+   Use case ends.
```

Comment:
Based on this, `Use case ends` is on the same line as `... a list of guests`. I noticed this issue in several other use cases too.

```suggestion
   <br>Use case ends.
```